oeis.py

```diff
@@ -8,11 +8,12 @@
 from typing import Collection, Dict, List, Callable
 import sys
 import os
-
 import numpy as np
 import matplotlib.pyplot as plt
 
 
+
+
```

Comment:
4 blanks lines are a bit too much here, if you run black I bet you'll end up with one or two, which is enough.